app/api/services/tasksmanager/specs/ExternalDummyService.ts

```diff
@@ -18,45 +18,65 @@ export class ExternalDummyService {
 
   currentTask: string | undefined;
 
-  materials: string[] = [];
+  materials: any[] = [];
 
   files: Buffer[] = [];
 
   filesNames: String[] = [];
 
-  results: object | undefined;
+  results: any | undefined;
 
   redisClient: RedisClient | undefined;
 
   fileResults: string | undefined;
 
   private readonly serviceName: string;
 
-  constructor(port = 1234, serviceName = 'dummy') {
+  materialsDataPartams: any;
```

Comment:
Typos in the variable names (lines 35-41).

Reply:
done